src/ontology/config/context.json

```diff
@@ -31,6 +31,8 @@
 		"NCBITaxon": "http://purl.obolibrary.org/obo/NCBITaxon_",
 		"ICD10CM": "http://purl.bioontology.org/ontology/ICD10CM/",
 		"ICD10WHO": "http://apps.who.int/classifications/icd10/browse/2010/en#/",
+		"icd11.foundation": "http://id.who.int/icd/entity/",
```

Comment:
## FYI: `config/context.json`
- [ ] Review & mark resolved @matentzn @twhetzel 

I needed to update this file as well. I found that docs about it were missing, so I made a PR for that:
- #470

I think this is a file that probably should be programmatically generated. I updated:
- https://github.com/monarch-initiative/mondolib/issues/8